test/images/agnhost/net/nat/closewait.go

```diff
@@ -36,10 +36,6 @@ import (
 	"k8s.io/kubernetes/test/images/agnhost/net/common"
 )
 
-// leakedConnection is a global variable that should leak the active
-// connection assigned here.
-var leakedConnection *net.TCPConn
```

Comment:
The comment make it sound like this is intentional, although I'm not sure this is actually going to do what was intended. In either case, I don't think the solution is just to delete this.

Reply:
Thanks, updated to ignore the staticcheck failure for now.